eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -31,23 +33,23 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="PublishTrimmed" DependsOnTargets="RestoreProject;UpdateRuntimePack;Publish" />
+  <Target Name="PublishTrimmed" DependsOnTargets="ProcessFrameworkReferences;RestoreProject;UpdateRuntimePack;Publish" />
 
   <ItemGroup>
     <KnownFrameworkReference Include="Microsoft.NETCore.App"
-      DefaultRuntimeFrameworkVersion="5.0.0-rc.1.20451.14"
+      DefaultRuntimeFrameworkVersion="$(MicrosoftNETCoreAppVersion)"
       IsTrimmable="true"
-      LatestRuntimeFrameworkVersion="5.0.0-rc.1.20451.14"
+      LatestRuntimeFrameworkVersion="$(MicrosoftNETCoreAppVersion)"
       RuntimeFrameworkName="Microsoft.NETCore.App"
       RuntimePackNamePatterns="Microsoft.NETCore.App.Runtime.**RID**"
       RuntimePackRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86;ios-arm64;ios-arm;ios-x64;ios-x86;tvos-arm64;tvos-x64;android-arm64;android-arm;android-x64;android-x86;browser-wasm"
       TargetFramework="net6.0"
       TargetingPackName="Microsoft.NETCore.App.Ref"
-      TargetingPackVersion="5.0.0-rc.1.20451.14" />
+      TargetingPackVersion="$(MicrosoftNETCoreAppVersion)" />
 
     <KnownAppHostPack Include="Microsoft.NETCore.App" 
       AppHostPackNamePattern="Microsoft.NETCore.App.Host.**RID**"
-      AppHostPackVersion="5.0.0-rc.1.20451.14"
+      AppHostPackVersion="$(MicrosoftNETCoreAppVersion)"
       AppHostRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86"
       TargetFramework="net6.0" />
```

Comment:
Any idea why this is hardcoded to net6.0 and not using the shared property?

Reply:
I can use `TargetFramework`... the shared property is not defined in this scope as this is only used on the generated projects which do't import this.